src/Model/CodeModelGo.cs

```diff
@@ -192,6 +192,10 @@ public IEnumerable<string> ModelImports
                 {
                     imports.UnionWith(CodeNamerGo.Instance.PageableImports);
                 }
+                if (ModelTypes.Any(mt => mt is CompositeTypeGo && mt.Cast<CompositeTypeGo>().NeedsCustomMarshalling))
+                {
+                    imports.Add(PrimaryTypeGo.GetImportLine("encoding/json"));

```

Comment:
I found it, in CompositeTypeGo.cs:216.  Method AddImports(), you can move this to there.

Reply:
No wonder I just search `"encoding/json"` and found nothing...